pythonequipmentdrivers/oscilloscope/Lecroy_WR8xxx.py

```diff
@@ -755,6 +770,52 @@ def get_channel_label(self, channel: int) -> str:
 
         return ' '.join(response.strip().split()[1:])
 
+    def set_channel_label_position(self, channel: int, position=0) -> None:
+        """
+        set_channel_label_position(channel, position)
+
+        updates the text label position on a channel specified by "channel"
+        with the position given in "position".
+
+        Args:
+            channel (int): channel number to update label of.
+            position (int): position trigger relative to place label
+        """
+
+        q_str = (f"""vbs 'app.acquisition.C{channel}.LabelsPosition = """ +
+                 f""""{position}" '""")
+        self.instrument.write(q_str)
+
+    def set_channel_label_view(self, channel: int, view: str = 'ON') -> None:
```

Comment:
I see that "view" is just as a boolean, can you just rewrite this to use a boolean? It would make it easier to use as you don't have to remember specific strings to pass it. Update the docstring accordingly with it.

Reply:
updated docstring, changed view to boolean. Noted in docstring the position reset behavior of the command